src/main/java/reposense/parser/CsvParser.java

```diff
@@ -37,26 +41,29 @@
             + "Content: %s\n"
             + "Error: %s";
     private static final String MESSAGE_EMPTY_CSV_FORMAT = "The CSV file, %s, is empty.";
-    private static final String MESSAGE_WRONG_HEADER_SIZE = "Wrong number of columns in header of CSV file, %s. \n"
-            + "Number of columns in header: %d\n"
-            + "Expected number of columns: %d";
+    private static final String MESSAGE_MANDATORY_HEADER_MISSING = "Required column header, %s, not found in "
+            + "CSV file, %s";
+    private static final String MESSAGE_DUPLICATE_COLUMN_HEADER = "Column header. %s, appears twice in CSV file, %s. "
+            + "The rightmost occurrence will be used.";
+    private static final String MESSAGE_COLUMNS_RECOGNIZED = "Parsed header of CSV file, %s, and recognized columns: "
+            + "%s";
     private static final String MESSAGE_ZERO_VALID_CONFIGS = "No valid configurations in the %s.";
 
     private Path csvFilePath;
-    private int expectedHeaderSize;
+    private int headerSize;
```

Comment:
This is used only once, I think we should keep it inside the `validateHeader` function as a local variable.

Reply:
I have changed `headerSize` to a local variable.